collections.go

```diff
@@ -25,8 +25,10 @@ func (cl *CollectionsClient) List() *CollectionsListClient {
 	return newCollectionListClient(cl.sl)
 }
 
+// Create returns a client for creating a collection.
+// API reference: https://www.getoutline.com/developers#tag/Collections/paths/~1collections.create/post
 func (cl *CollectionsClient) Create(name string) *CollectionsCreateClient {
-	return newCollectionsCreateClient(cl.sl, name)
+	return newCollectionsCreateClient(cl.sl, collectionsCreateParams{Name: name})
```

Comment:
if you agree with other comments then original code is fine

Reply:
Fixed here https://github.com/ioki-mobility/go-outline/pull/29/commits/bee03ffdc515c91f871b68f2a7b78d4085f2b399